### PR TITLE
docs: add guidance for handling CRLF on Windows

### PR DESCRIPTION
**Pull request by jennings**
Attempts to add prescriptive guidance for dealing with LF line endings on Windows.

# Checklist

If applicable:
- [ ] I have updated `CHANGELOG.md`
- [X] I have updated the documentation (README.md, docs/, demos/)
- [ ] I have updated the config schema (cli/src/config-schema.json)
- [ ] I have added tests to cover my changes

### CODE REVIEW

**Review by steveklabnik — APPROVED** [review]
This looks great to me!

File: docs/windows.md [hunk 1]
```diff
@@ -5,10 +5,31 @@ users should be aware of.
...
-Jujutsu does not honor `.gitattributes` and does not have a setting like Git's
-`core.autocrlf`. This means that line endings will be checked out exactly as
-they are committed and committed exactly as authored. This is true on all
-platforms, but Windows users are most likely to miss CRLF conversion.
+Jujutsu does not currently honor `.gitattributes` and does not have a setting
+like Git's `core.autocrlf`. This means that line endings will be checked out
+exactly as they are committed and committed exactly as authored. This is true on
+all platforms, but Windows users are most likely to miss CRLF conversion.
+
+If you Git repository expects Windows users to have `core.autocrlf` set to
```
Comment (PhilipMetzger) [inline docs/windows.md:13]: Typo: "If you" => "If your Git repository ..."

**Review by PhilipMetzger — APPROVED** [review]
LG

File: docs/windows.md (hunk 1 above)
Comment (jennings) [reply docs/windows.md]: Whoops, fixed.

**Pull request merged**